### ISSUE TITLE
eliminate context menu on file explorer

### ISSUE THREAD

**Issue by peller** [maqetta/maqetta#499]
It occurs to me that there's no need for this context menu.  Some elements are already on the toolbar, others (like upload, new folder, etc.) likely should be. Many don't even depend on the context, making them even less appropriate, and the user really has no idea those commands are even there.  I'd also argue that context menus aren't very webby and should be avoided in our app.

@JonFerraiolo @jmockler

**Comment by jhpedemonte** [comment]
I agree, to an extent, with the notion that **_context menus aren't very webby**_.  I can't really think of any web apps that consistently use context menus -- maybe some of the Google apps, like Docs (Mail doesn't seem to use them).

Of the 5 items in the context menu, only 2 of them (**Delete Resource** and **Download**) are specific to the item the user has clicked on.  The other 3 (**New File**, **New Folder**, **Upload**) don't really belong in the context menu and are better handled as toolbar actions.

The first 2 could also be handled on the toolbar without much problem.  We could also expand the delete action -- if a user clicks (highlights) a file or folder in the Files tree, we can display a red **X** on the right of that file.  But not really necessary.

So, to go along with @peller, I don't think the actions we currently have in the context menu really justify having a context menu...

**Comment by peller** [comment]
see #884

-> Prio high

**Comment by JonFerraiolo** [comment]
@jhpedemonte -> M8 Prio-Med - not an M8 ship-stopper
